wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Commands.java

```diff
@@ -159,10 +173,20 @@ public static Command select(Map<Object, Command> commands, Supplier<Object> sel
    * @see SequentialCommandGroup
    */
   public static Command sequence(Command... commands) {
-    return new SequentialCommandGroup(commands);
+    return sequence(true, commands);
   }
 
-  // Command Groups
+  /**
+   * Runs a group of commands in series, one after the other.
+   *
+   * @param requireUnion whether the group should require the union of its member commands' requirements
+   * @param commands the commands to include
+   * @return the command group
+   * @see SequentialCommandGroup
+   */
+  public static Command sequence(boolean requireUnion, Command... commands) {
```

Comment:
This is rather problematic -- a very important semantic difference shouldn't be controlled by a boolean parameter.


A `proxySequence` factory would be better.

Reply:
agree